### DISCUSSION TITLE
SARS-CoV-2 Spike mutations

### DISCUSSION THREAD

**Discussion by martyLY** [brianhie/viral-mutation#40] [Q&A]
Hello Dr. Hie,

Thanks for share your amazing work!

But I don't get why not consider delete and insert mutations of SARS-CoV-2 Spike?
(link: https://github.com/brianhie/viral-mutation/blob/81c80d41671670eb58cc46e957a1b0c4bf14856a/bin/cov.py#L273)

**Comment by brianhie** [comment]
Hi @martyLY, thanks for reaching out, as noted in the paper, our approach does not naturally extend to insertions and deletions. Though there are language models that can model indels in a more [principled way](https://arxiv.org/abs/2205.13760).

**Reply by martyLY** [reply]
Thank your reply very much ~~~~
